cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_ionic_client_app/templates/src/assets/i18n/it.json.ftl

```diff
@@ -1,65 +1,65 @@
-{
-  "buttons": {
-    "login": "Send",
-    "logout": "logout",
-    "menu": "Menu",
-    "back": "back",
-    "Add": "Add",
-    "Modify": "Modify",
-    "Inspect": "Inspect",
-    "Delete": "Delete",
-    "Filter":"Filter"
-  },
-  "header": {
-    "title": "Ionic",
-    "error": "LOGIN ERROR"
-  },
-   "alert":{
-    "title":"Error",
-    "subtitle":"Wrong User and/or password",
-    "dismiss":"Close"
-  },
-  "${variables.component?uncap_first}": {
-    "title": "${variables.component} Title",
-    "description": "This is the basic template for ionic",
-    "${variables.etoName?uncap_first}": {
-     
-       <#list model.properties as property>
-    "${property.name}":"${property.name}_IT",
-    </#list>
-    "commonbuttons":{
-      "send": "Send",
-      "dismiss":"Close"
-    },
-    "operations": {
-       "filter":{
-        "title":"Search form",
-        "message":"Use this form to search for a item",
-        "clear": "Clear Filter"
-      }, 
-       "add":{
-        "title":"Add form",
-       "message":"This form is used to add new items"
-      },
-       "modify":{
-        "title":"Modify form",
-       "message":"Here you can modify the selected user"
-      },
-      "delete":{
-        "title":"Delete Confirmation Alert",
-        "message":"Do you want to delete the selected element?",
-        "dismiss":"No, cancel",
-        "confirm":"Yes"
-      }
-    }
-    }
-  },
-  "login": {
-    "user": "User",
-    "username": "Username",
-    "password": "Password",
-    "errorMsg": "Wrong username or password"
-  },
-  "example":"example",
-  "Home":"Home"
+{
+  "buttons": {
+    "login": "Send",
+    "logout": "logout",
+    "menu": "Menu",
+    "back": "back",
+    "Add": "Add",
+    "Modify": "Modify",
+    "Inspect": "Inspect",
+    "Delete": "Delete",
+    "Filter":"Filter"
+  },
+  "header": {
+    "title": "Ionic",
+    "error": "LOGIN ERROR"
```

Comment:
I think you forgot to pull the latest changes from CobiGen master. This file is outdated, it should not be modified.